Optionally/Options.cs

```diff
@@ -34,6 +34,15 @@ public interface IOption<T>
         /// <param name="filter">Predicate used to test Option</param>
         /// <returns>If Option is Some and the value fulfills the filter, then Some. Otherwise None</returns>
         IOption<T> Where(Func<T, bool> filter);
+
+        /// <summary>
+        /// Performs a Fuunction on the current Options
```

Comment:
Small typo, believe you meant _Function_ not _Fuunction_

Reply:
Changed and commited